services/common/src/main/java/org/collectionspace/services/common/storage/jpa/JpaStorageUtils.java

```diff
@@ -375,18 +381,12 @@ public static AccountRole getAccountRoles(String csid)
 			Query q = em.createQuery(queryStr);
 			resultList = q.getResultList().iterator();
 
-			List<RoleValue> roleValues = new ArrayList<RoleValue>();
-
-			while (resultList.hasNext()) {
-				AccountRoleRel accountRolRel = (AccountRoleRel) resultList.next();
-				RoleValue roleValue = AuthorizationRoleRel.buildRoleValue(accountRolRel);
-
-				if (roleValue != null) {
-					roleValues.add(roleValue);
+			if (resultList.hasNext()) {
+				List<RoleValue> roleValues = new ArrayList<RoleValue>();
+				while (resultList.hasNext()) {
+					AccountRoleRel accountRolRel = (AccountRoleRel)resultList.next();
+					roleValues.add(AuthorizationRoleRel.buildRoleValue(accountRolRel));
```

Comment:
We lost a null check on `AuthorizationRoleRel.buildRoleValue(accountRolRel)`. Is that intentional?

Reply:
Not intentional.  Looks like old code from before a change @ray-lee made several years ago.